test/types/index.ts

```diff
@@ -0,0 +1,6 @@
+import Fastify from 'fastify'
+import fastifyExpress = require('../..')
```

Comment:
You should be able to avoid require by following this doc: https://www.fastify.io/docs/latest/TypeScript/#creating-type-definitions-for-a-fastify-plugin.

Reply:
ah, nice. Will do!

The docs are a bit dated tho. `FastifyPlugin` is deprecated (`import { FastifyPlugin } from 'fastify'`)